ios/FluentUI/BottomSheet/BottomSheetViewController.swift

```diff
@@ -21,6 +21,8 @@ public protocol BottomSheetViewControllerDelegate: AnyObject {
 
 public class BottomSheetViewController: UIViewController {
     @objc public weak var delegate: BottomSheetViewControllerDelegate?
+    /// bottom sheet's view height in collapsed mode.
+    @objc public var collapsedHeight: CGFloat = 200
```

Comment:
we should also have didSet{} which if the value change it will call your updateHeight()